### PR TITLE
Set current development revisions to meta-layers and components

### PR DESCRIPTION
**Pull request by iartemenko**
Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

**Comment by lorc** [comment]
s/meeta/meta in the commit message.

Also, I would like to see more elaborated commit message. Something like "Switch from ces2018 to development branches in xt projects, because meta-xt-prod-devel layer is primarily used for development"

**Comment by iartemenko** [comment]
Done

File: recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend [hunk 1]
```diff
@@ -3,7 +3,7 @@ SRCREV = "${AUTOREV}"
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 SRC_URI_append = " \
-    repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_ces2018/doma.xml;scmdata=keep \
+    repo://git@gitpct.epam.com/epmd-aepr/android_manifest.git;protocol=ssh;branch=master;manifest=doma.xml;scmdata=keep \
```
Comment (andr2000) [inline recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend]: why ssh instead of https?

Comment (andr2000) [inline recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend]: IMO, commit message should also note that Android's xml moves from xen-troops to gitlab

Comment (iartemenko) [reply recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend]: Done

**Comment by andr2000** [comment]
from *gitlad*

**Pull request merged**